streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -661,6 +673,131 @@ public void statefulTopologyShouldCreateStateDirectory() throws Exception {
         startStreamsAndCheckDirExists(topology, asList(inputTopic, globalTopicName), outputTopic, true);
     }
 
+    @SuppressWarnings("unchecked")
+    @Test
+    public void shouldInitializedUserSerdes() {
```

Comment:
The verb following 'should' would be in active tense.

consider naming this test: userSerdesShouldBeInitialized